src/main/webapp/script.js

```diff
@@ -98,13 +106,11 @@ function accessFunction() {
     return;
   }
 
-  makeFormDisappear();
-  
   config.projectID = projectId;
   config.bucketName = bucketName;
   config.objectName = objectName;
 
-  initBody();
```

Comment:
I may have missed this, but where is initBody now called?

Reply:
line 27

Reply:
I see! Completely missed that 😅